tests/python/pants_test/backend/python/tasks/python_task_test_base.py

```diff
@@ -65,9 +65,11 @@ def create_python_binary(self, relpath, name, entry_point, dependencies=(), prov
         {dependencies}
       ],
       {provides_clause}
+      {shebang_clause}
     )
     """).format(name=name, entry_point=entry_point, dependencies=','.join(map(repr, dependencies)),
-                provides_clause='provides={0},'.format(provides) if provides else ''))
+                provides_clause="provides={0},".format(provides) if provides else '',
+                shebang_clause="shebang='{0}',".format(shebang) if shebang else ''))
```

Comment:
Ditto - and you can use `shebang={!r}` to realize this.

Reply:
Nice, didn't know that one.